exchange/utils_test.go

```diff
@@ -4288,6 +4289,7 @@ func TestCleanOpenRTBRequestsActivitiesFetchBids(t *testing.T) {
 			expectedUserYOB:      1982,
 			expectedUserLat:      123.456,
 			expectedDeviceDIDMD5: "some device ID hash",
+			expectedSourceTID:    "61018dc9-fa61-4c41-b7dc-f90b9ae80e87",
```

Comment:
Nitpick: Is there a need to use a guid as a test string? Could we instead use something simpler, such as "1" or "anyTID"? Specific values which are not immediately obvious as fake data may be misinterpreted as conveying a special meaning.